cmd/podman/stats.go

```diff
@@ -215,18 +223,29 @@ func (i *statsOutputParams) headerMap() map[string]string {
 }
 
 func combineHumanValues(a, b uint64) string {
+	if a == 0 && b == 0 {
```

Comment:
is it possible for only a or only b to be zero?

Reply:
I have no idea, but in that case I think we would want to show the values, since that would indicate that the container was running.

"--" seems to indicate that the container is not running, where 0% would indicate it is running, but using no  resources.